modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS). You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
```

Comment:
If this PR is time sensitive, I don't think this question needs to be answered now. But I do wonder why I'd want to forward logs to Cloudwatch in addition to or instead of OpenShift Logging Elasticsearch? 

Right now, I'm not sure of the benefit of Amazon CloudWatch over Elasticsearch or vice versa. Granted the benefit of both could have been communicated in a previous doc. 

Still, I wonder if you could delete the second sentence. Seeing the title `Forwarding logs to Amazon Cloudwatch` I think "that's the action I'll be doing." So I don't know why I even need to consider OpenShift Logging-managed Elasticsearch log store. Unless I know the benefit of it, I don't want to consider it. Basically, this second sentence distracts me from the task at hand which is to forward logs to Amazon CloudWatch.

Reply:
@jc-berger This entire chapter on forwarding logs to 3rd party systems is about how to, we don't go into why you'd want to do that on any of them. So in that sense the new text on CloudWatch just keeps with the tradition. We're assuming you know why you need to forward your logs, we're telling you how.

Reply:
Good questions, Jake. We added this statement to all the "forwarding" topics to address customer questions about the role of the built-in OpenShift Elasticsearch logging service vs. their external logging server. Fixing this issue at a deeper level will require restructuring the Logging docs, which is out of scope here.